src/js/tools/transform/Rotate.js

```diff
@@ -1,15 +1,15 @@
 (function () {
   var ns = $.namespace('pskl.tools.transform');
 
-  ns.Rotate = function () {
+  ns.Rotate = function (i18n) {
     this.toolId = 'tool-rotate';
-    this.helpText = 'Counter-clockwise rotation';
+    this.helpText = i18n.counterClockwiseRotation();
 
-    this.tooltipDescriptors = [{key : 'alt', description : 'Clockwise rotation'}];
+    this.tooltipDescriptors = [{key : 'alt', description : i18n.counterClockwiseRotationDescriptorRotation()}];
     if (Constants.ENABLE_MULTIPLE_LAYERS) {
-      this.tooltipDescriptors.push({key : 'ctrl', description : 'Apply to all layers'});
+      this.tooltipDescriptors.push({key : 'ctrl', description : i18n.applyToAllLayersDescriptor()});
```

Comment:
Consider adding the tool name to the beginning of the string key. For example `rotateTransformApplyToAllLayersDescriptor`

Doing this makes it so it is unlikely there is string key collisions. For example, this string key `applyToAllLayersDescriptor` is also used in the above Flip.js. We want different string keys for these files because right now they might have the same English string, but in the future they might not.

Another benefit of adding more context to the string key is that it makes it easier to figure out where the string is being used just by looking at the key.